### PR TITLE
CH20 SONU KUMAR KUSHWAHA -> IMPLEMENTED THE STACK VIA QUEUE AND VICE …

### PR DESCRIPTION
**Pull request by sonukushwaha403**
…VERSA

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by sonukushwaha403** [comment]
CH20 ISSUE #412 is done
sonu kumar kushwaha

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@sonukushwaha403 I think it is similar to https://www.geeksforgeeks.org/queue-using-stacks/, please change it. Also, whenever you add codes do update the readme accordingly.

File: Python/ds/stack_via_queue.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Program to implement a stack using
```
Comment (iamrajiv) [inline Python/ds/stack_via_queue.py]: @sonukushwaha403 similar to this https://www.geeksforgeeks.org/implement-stack-using-queue/

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/ds/stack_via_queue.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Program to implement a stack using  
+# two queue
+from queue import Queue
```
Comment (vatsalkesarwani12) [inline Python/ds/stack_via_queue.py]: You have to use the only list for writing your code as clearly mentioned no inbuilt function

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
do the changes

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Plagiarized

**Comment by sonukushwaha403** [comment]
please review is have done changes

**Comment by siddharth25pandey** [comment]
@sonukushwaha403 Refer [this](https://github.com/TesseractCoding/NeoAlgo/pull/431#pullrequestreview-468036613)

**Pull request closed without merging**